apps/cloudinary2/src/locations/Field/index.tsx

```diff
@@ -18,22 +18,26 @@ const Field = () => {
   const sdk = useSDK<FieldAppSDK<AppInstallationParameters>>();
   useAutoResizer({ absoluteElements: true });
 
-  const [assets = [], setAssets] = useFieldValue<CloudinaryAsset[]>(sdk.field.id, sdk.field.locale)
+  const [assets = [], setAssets] = useFieldValue<CloudinaryAsset[]>(sdk.field.id, sdk.field.locale);
+  const assetsIsNotNull = assets !== null;
```

Comment:
Just curious here, it looks like if useFieldValue returns `null` or `undefined`, the destructuring assignment would ensure that `assets` is set to an empty array so assets should never be null right? It might be redundant 🤔 or am I off base here?

Reply:
I had a similar line of thinking and questions here, and this also connects to my other comment. If assets is null this later disables the ability to add assets. Not sure if that's what we want? If not, your suggestion sounds like a good one (just fall back to an empty array when null or undefined)